mathjax2/legacy/jax/element/MmlNode.js

```diff
@@ -48,6 +48,7 @@
             }
           }
         }
+        if (this['class']) node.attributes.set('class',this['class']);
```

Comment:
Note:  This is in the mathjax2 legacy code, which (a) will not be in the final version, and (b) does not use the same formatting restrictions that are in the v3 code.  So I think we can leave this one.